src/main/java/com/aliyuncs/fc/model/TimeTriggerConfig.java

```diff
@@ -0,0 +1,48 @@
+package com.aliyuncs.fc.model;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Time Trigger config
+ */
+public class TimeTriggerConfig {
+
+    @SerializedName("cronExpression")
+    private String cronExpression;
+
+    @SerializedName("payload")
+    private String payload;
+
+    @SerializedName("enable")
+    private boolean enable;
```

Comment:
This should be Boolean type so the default value is null and won't be included in the payload if it's not set.